### PR TITLE
point seeded tracking regions

### PR DESCRIPTION
**Pull request by mtosi**
this is the first (and probably really rude) version a new tracking region creator
starting from a point in eta-phi plane (the r component is set to 1)
@VinInn @rovere

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
Why do we need existAs in new code?
There is no fillDescriptions method.
Is the base class already a thread-safe EDProducer?

**Comment by VinInn** [comment]
Thanks, actually we are interested to use it offline as well

File: RecoTauTag/HLTProducers/src/SealModule.cc [hunk 1]
```diff
@@ -32,6 +33,7 @@
 DEFINE_EDM_PLUGIN(TrackingRegionProducerFactory, TauRegionalPixelSeedGenerator, "TauRegionalPixelSeedGenerator");      
 DEFINE_EDM_PLUGIN(TrackingRegionProducerFactory, TrackingRegionsFromBeamSpotAndL2Tau, "TrackingRegionsFromBeamSpotAndL2Tau");
 DEFINE_EDM_PLUGIN(TrackingRegionProducerFactory, CandidateSeededTrackingRegionsProducer, "CandidateSeededTrackingRegionsProducer");
+//DEFINE_EDM_PLUGIN(TrackingRegionProducerFactory, PointSeededTrackingRegionsProducer, "PointSeededTrackingRegionsProducer");
```
Comment (Martin-Grunewald) [inline RecoTauTag/HLTProducers/src/SealModule.cc]: Please just restore the file as it was before this PR (adding commented lines is useless and requires HLT signature):
RecoTauTag/HLTProducers/src/SealModule.cc

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by cvuosalo** [comment]
+1

For #9149 0cb9bf74b1c3e94131cf5ac3bd9508a59be77c7b

This PR adds a new tracking region creator. The new code is not called by standard sequences yet, so it should not change monitored quantities.

The new code is satisfactory, and Jenkins [tests](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-9149/4833/summary.html) against baseline CMSSW_7_5_X_2015-05-18-1100 show no significant differences, as expected.

**Comment by davidlange6** [comment]
+1

**Pull request merged**